ProjectsManagingSystem/Services/Project/ProjectService.cs

```diff
@@ -103,6 +103,49 @@ public IEnumerable<ProjectTaskResponseDto> GetTasks(int id)
         return result;
     }
 
+    public IEnumerable<ProjectTaskResponseDto> GetMemberProjectTask(int id, int memberId)
+    {
+
+        var project = _dbContext
+            .Projects
+            .Include(t => t.Tasks)
+            .FirstOrDefault(x => x.Id == id);
+
+        if (project == null)
+        {
+            // Return an empty list if the project is not found
+            return new List<ProjectTaskResponseDto>();
+        }
+        var tasks = project.Tasks
+            .Where(t => t.MemberId == memberId)
+            .ToList();
+
+        var result = _mapper.Map<List<ProjectTaskResponseDto>>(tasks);
```

Comment:
its more efficient to get all members from project and then for a specific ID return MemberTasks:
project.include(members) -> member. tasks(t.id = id)